socli/tests/test_socli.py

```diff
@@ -44,6 +44,7 @@ def test01_socliParse(self):
         self.assertFalse(namespace.sosearch)
         self.assertFalse(namespace.tag)
         self.assertFalse(namespace.user)
+        self.assertTrue(namespace.open-url==[])
```

Comment:
same here and all the other tests below, should be `open_url` as the namespace attribute because `-` is minus in Python and even though the option name is `--open-url`, the dest is still `open_url`.

That's mainly why the latest Travis tests are failing